src/app/backend/replicasetpods.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+	"k8s.io/kubernetes/pkg/fields"
+	"k8s.io/kubernetes/pkg/labels"
+)
+
+// Information about a Container that belongs to a Pod.
+type PodContainer struct {
+	// Name of a Container.
+	Name string `json:"name"`
+
+	// Container state.
+	State string `json:"state"`
+
+	// Number of restarts.
+	RestartCount int `json:"restartCount"`
+}
+
+// List of pods that belongs to a Replica Set.
+type ReplicaSetPods struct {
+	// List of pods that belongs to a Replica Set.
+	Pods []ReplicaSetPodWithContainers `json:"pods"`
+}
+
+// Detailed information about a Pod that belongs to a Replica Set.
+type ReplicaSetPodWithContainers struct {
```

Comment:
yes I took this name to avoid collision inside the package. Can it stay like this ?